diff --git a/MediaBrowser.Model/Configuration/ServerConfiguration.cs b/MediaBrowser.Model/Configuration/ServerConfiguration.cs
index 52f7e53b81..b149e3251a 100644
--- a/MediaBrowser.Model/Configuration/ServerConfiguration.cs
+++ b/MediaBrowser.Model/Configuration/ServerConfiguration.cs
@@ -67,7 +67,7 @@ public class ServerConfiguration : BaseApplicationConfiguration
     /// <summary>
     /// Gets or sets a value indicating whether to enable prometheus metrics exporting.
     /// </summary>
-    public bool EnableMetrics { get; set; } = false;
+    public bool EnableMetrics { get; set; } = true;
 
     public bool EnableNormalizedItemByNameIds { get; set; } = true;
 
